background_scripts/commands.js

```diff
@@ -551,6 +562,11 @@ const commandDescriptions = {
   moveTabLeft: ["Move tab to the left", { background: true }],
   moveTabRight: ["Move tab to the right", { background: true }],
 
+  setZoom: ["Set zoom to the value given after the command, like setZoom 2.25", { background: true }],
```

Comment:
I wanted to add an example here to make the usage more clear, but if there is a better place for this or a better way to make the usage clear, we should remove it here.

Reply:
Assuming the zoom value must be given as part of the command definition, then documenting that here would be unusual, as this string is shown in the help dialog.

Where to document a command's options is an unsolved issue. I think the #2827 has the most substantial discussion about it. The [tips and tricks wiki page](https://github.com/philc/vimium/wiki/Tips-and-Tricks) contains various command options organized by use case.

For now, I'd suggest this docstring: "Set zoom level to a given value. E.g. `map zz setZoom 1.5`"